aux/vagrant-build/Vagrantfile

```diff
@@ -1,67 +1,32 @@
-  # There is no livecd-tools available in EL 7.0-7.3 - use Fedora instead
-  {:name => 'el7',      :libvirt => 'fm-centos70',   :image_name => /CentOS 7/, :default => true, :pty => true},
-]
-
-if ENV['box']
-  boxes << {:name => ENV['box'], :libvirt => ENV['box'], :image_name => ENV['box'], :os_user => ENV['box']}
-end
-
 SHELLARGS = []
 SHELLARGS << ENV['repoowner'] || ''
 SHELLARGS << ENV['branch'] || ''
 SHELLARGS << ENV['proxy_repo'] || ''
 
 Vagrant.configure("2") do |config|
-  boxes.each do |box|
-    config.vm.define box[:name], primary: box[:default] do |machine|
-      machine.vm.box = box[:name]
-      machine.vm.hostname = "foreman-#{box[:name]}.imagebuilderv2.theforeman.org"
-      machine.vm.provision :shell, :path => 'build_image.sh', :args => SHELLARGS
-
-      machine.vm.provider :libvirt do |p, override|
-        override.vm.box = "#{box[:libvirt]}"
-        override.vm.box_url = "http://m0dlx.com/files/foreman/boxes/#{box[:libvirt].sub(/^fm-/, '')}.box"
-        p.memory = 1024
-      end
+  config.ssh.username  = "vagrant"
+  config.ssh.pty = true
 
-      machine.vm.provider :rackspace do |p, override|
-        override.vm.box = 'dummy'
-        p.server_name = machine.vm.hostname
-        p.flavor = /1GB/
-        p.image = box[:image_name]
-        p.rackspace_region = :iad
-        override.ssh.pty = true if box[:pty]
-        #override.nfs.functional = false
-      end
+  config.vm.define "el7", primary: true do |machine|
+    machine.vm.hostname = "fdi-builder-vm"
+    machine.vm.provision :shell, :path => 'build_image.sh', :args => SHELLARGS
 
-      # ~/.vagrant.d/Vagrantfile will need
-      # config.ssh.private_key_path = "~/.ssh/id_rsa"          # private key for keypair below
-
-      config.vm.provider :openstack do |p, override|
-        override.vm.box = 'dummy'
-        p.server_name   = machine.vm.hostname
-        p.flavor        = /m1.tiny/
-        p.image         = box[:image_name] # Might as well use consistent image names
-        p.ssh_username  = box[:os_user]  # login for the VM
-
-        # ~/.vagrant.d/Vagrantfile will need
-        # p.username     = "admin"                             # e.g. "#{ENV['OS_USERNAME']}"
-        # p.api_key      = "secret"                            # e.g. "#{ENV['OS_PASSWORD']}"
-        # p.endpoint     = "http://openstack:5000/v2.0/tokens" # e.g. "#{ENV['OS_AUTH_URL']}/tokens"
-        # p.keypair_name = "my_key"                            # as stored in Nova
+    config.vm.provider :libvirt do |lv|
+      lv.server_name = machine.vm.hostname
+      lv.vm.box = "centos/7"
+      lv.memory = 1024
+    end
 
-        # You may need
-        # p.floating_ip = '172.20.10.160' # Must be hardcoded, cannot ask Openstack for an IP
-      end
+    # openstack needs credentials in ~/.vagrant.d/Vagrantfile and existence
+    # of a dummy box:
+    # vagrant box add dummy https://github.com/cloudbau/vagrant-openstack-plugin/raw/master/dummy.box
```

Comment:
The dummy box is no longer needed with the provider we use (https://github.com/ggiamarchi/vagrant-openstack-provider).